examples/glue_benchmark/glue.py

```diff
@@ -11,10 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import csv
+
+# DISCLAIMER:This script only supports GLUE/mrpc (for now). #
```

Comment:
You can use the standard head format and include a one-sentence description of this file. For example:

""" GLUE benchmark script to test model performance.

To run the script, use this command:
```
{Put command here}
```

Disclaimer: This script only supports GLUE/mrpc (for now).
"""